### ISSUE TITLE
CS231N 7강 - Training Neural Networks II

### ISSUE THREAD

**Issue by fool8474** [IPCVLabEdu/flipped_learning#29]
1. 2차 근사를 활용할때, Hessian matrix를 이용한다고 하는데, 이를 어떤식으로 이용해서 Optimization의 역할을 할 수 있게 하는건가요? 그리고 Optimization을 할 때 2차 근사를 사용하면 이점이 있나요?
![image](https://user-images.githubusercontent.com/28250401/53151688-2e361000-35f7-11e9-90e7-1b9a39e5117d.png)

2. Batch Normalization이나 Data argumentation 등의 설명을 할때 stochasticity라는 단어가 종종 나옵니다. 검색하니 확률론이라는 번역이 나오는데, 너무 추상적인 단어라서 잘 이해가 안 갑니다. 이 단어를 인공지능 분야에서는 어떻게 받아들여야하나요?
ex) train time에서는 각 데이터에 대해 어떻게 정규화시킬지 stochasticity가 존재했으나 test time에서는 정규화를 mini batch단위가 아닌 global 단위로 수행, stochasticity를 평균화한다.

3. 전이학습에서 이야기하는 fine-tune이 어떤것인가요?
